src/style.css

```diff
@@ -0,0 +1,63 @@
+* {
```

Comment:
if you need to reset indentations, do it for each of the block you need

Reply:
done